custom_components/amplipi/media_player.py

```diff
@@ -1063,10 +1049,15 @@ async def async_play_media(self, media_type, media_id, **kwargs):
 
     @property
     def extra_state_attributes(self):
+        zones = self._get_zone_ids() if self._group is not None else None
+
+        # amplipi_zone_id and amplipi_zone_id are used by the group card of AmpliPi-HomeAssistant-Card to select related zone entities so they can be listed individually on the card
```

Comment:
amplipi_zone_id and amplipi_zone_id ??

Reply:
Is it necessary to always populate these fields? Could we only populated amplipi_zones when it is a group?